src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -110,6 +68,49 @@
 import org.xmlpull.v1.XmlPullParser;
 import org.xmlpull.v1.XmlPullParserException;
 
+import java.io.ByteArrayInputStream;
```

Comment:
This was rearranged by IntelliJ. I think it's a good change.

Reply:
It does that all the time, and I love it :)

I think we should always defer on IntelliJ the reformatting of the file, so here's my explicit :+1:

Using automatic reformatting makes collaboration easier!